spring-boot-project/spring-boot-starters/README.adoc

```diff
@@ -230,7 +230,7 @@ do as they were designed before this was clarified.
 | https://github.com/fonimus/ssh-shell-spring-boot
 
 | https://github.com/savantly-net/sprout-platform[Sprout Platform]
-| https://github.com/savantly-net/sprout-platform/tree/master/spring/sprout-spring-boot-starter
+| https://github.com/savantly-net/sprout-legacy-starter
```

Comment:
I think this should be updated to https://github.com/savantly-net/sprout-platform/tree/master/backend/starters/sprout-spring-boot-starter instead.

Reply:
You are right, done.